content/id/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -0,0 +1,196 @@
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Berkas konfigurasi
+
+Untuk membuat _load balancer_ eksternal, tambahkan baris dibawah ini ke
+[berkas konfigurasi servis](/docs/concepts/services-networking/service/#loadbalancer) kamu:
+
+```yaml
+    type: LoadBalancer
+```
+
+Berkas konfigurasi kamu mungkin terlihat seperti ini:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-service
+spec:
+  selector:
+    app: example
+  ports:
+    - port: 8765
+      targetPort: 9376
+  type: LoadBalancer
+```
+
+## Menggunakan kubectl
+
+Kamu dapat membuat servis dengan perintah `kubectl expose` dan
+_flag_ `--type=LoadBalancer`:
+
+```bash
+kubectl expose rc example --port=8765 --target-port=9376 \
+        --name=example-service --type=LoadBalancer
+```
+
+Perintah ini membuat servis baru dengan menggunakan pemilih yang sama dengan
+sumber daya yang dirujuk (dalam hal contoh diatas, pengendali replikasi bernama `example`).
+
+Untuk informasi lebih lanjut, termasuk opsi _flag_, mengacu kepada
+[referensi `kubectl expose`](/docs/reference/generated/kubectl/kubectl-commands/#expose).
+
+## Menemukan alamat IP kamu
+
+Kamu dapat menemukan alamat IP yang telah dibuat untuk servis kamu dengan mendapatkan
+informasi servis melalui `kubectl`:
+
+```bash
+kubectl describe services example-service
+```
+
+yang seharusnya menghasilkan keluaran seperti ini:
+
+```bash
+    Name:                   example-service
+    Namespace:              default
+    Labels:                 <none>
+    Annotations:            <none>
+    Selector:               app=example
+    Type:                   LoadBalancer
+    IP:                     10.67.252.103
+    LoadBalancer Ingress:   192.0.2.89
+    Port:                   <unnamed> 80/TCP
+    NodePort:               <unnamed> 32445/TCP
+    Endpoints:              10.64.0.4:80,10.64.1.5:80,10.64.2.4:80
+    Session Affinity:       None
+    Events:                 <none>
+```
+
+Alamat IP tercantum di sebelah `LoadBalancer Ingress`.
+
+{{< note >}}
+Juka kamu menjalankan servis dari Minikube, kamu dapat menemukan alamat IP dan porta yang ditetapkan dengan:
+{{< /note >}}
+
+```bash
+minikube service example-service --url
+```
+
+## Preservasi IP sumber klien
+
+Dikarenakan implementasi fitur ini, sumber IP yang terlihat pada _container_
+target *bukan sebagai sumber IP asli* dari klien. Untuk mengaktifkan
+preservasi IP klien, bidang berikut dapat dikonfigurasikan didalam
+spek servis (mendukung lingkungan GCE/Google Kubernetes Engine):
+
+* `service.spec.externalTrafficPolicy` - menunjukan jika Service menginginkan untuk merute lalu lintas
+eksternal ke titik akhir _node-local_ atau _cluster-wide_. Terdapat dua opsi yang tersedia:
+Cluster (_default_) dan Local. Cluster mengaburkan sumber IP klien dan mungkin menyebabkan
```

Comment:
should be with backtick since Cluster and Local are values of field `externalTrafficPolicy`
```suggestion
`Cluster` (bawaan) dan `Local`. `Cluster` mengaburkan sumber IP klien dan mungkin menyebabkan
```